### PR TITLE
Updated readme instructions for storage

### PR DESCRIPTION
**Pull request by maneesht**
Updated README to include CORS setup information to run Storage tests.

### CODE REVIEW

File: packages/storage/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+cors.json
```
Comment (tonyjhuang) [inline packages/storage/.gitignore:1]: remove?

**Review by tonyjhuang — APPROVED** [review]
LGTM except one comment

File: packages/storage/.gitignore (hunk 1 above)
Comment (maneesht) [reply packages/storage/.gitignore]: Done.

**Review by hsubox76 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -113,6 +113,26 @@ database. When prompted to select the set of initial security rules, select
 any option (e.g. "Start in Production Mode") since these permission settings
 will be overwritten below.
 
+#### Storage Setup
+
+Visit the "Storage" section of the console and create a storage bucket. In
```
Comment (dwyfrequency) [inline README.md:118]: The double quotes look good to me but it could qualify for backticks. @hsubox76 what do you think here?

Comment (maneesht) [reply README.md:118]: I'm happy to change it, but the rest of the file uses double quotes, so if we change one, we should probably change the rest

**Review by dwyfrequency — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (dwyfrequency) [reply README.md:118]: in that case, double quotes is good

**Pull request merged**